packages/synapse-core/src/session-key/permissions.ts

```diff
@@ -1,5 +1,5 @@
 import { TypedData } from 'ox'
-import { keccak256, stringToHex } from 'viem'
+import { type Hex, keccak256, stringToHex } from 'viem'
```

Comment:
type imports need to be separate I think for the linter, currently linter isn't running on synapse-core in GHA but this will become a problem later I think, pull this up to a separate import